### DISCUSSION TITLE
docker

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1396] [General]
我自己修改了项目当中的一些内容，之后封装docker，出现了一个问题，
curl http://localhost:3210/
curl: (56) Recv failure: Connection reset by peer

响应中断了

而且他是隔一段时间出现，没有规律，为什么？一会可以用，一会就不行

老师，这是什么原因？

这是代理的报错
2024/02/27 10:10:37 [error] 580955#580955: *227 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 61.48.41.22, server: aiex.mosslab.hqmoss.com, request: "GET /sw.js HTTP/1.1", upstream: "http://127.0.0.1:3210/sw.js", host: "aiex.mosslab.hqmoss.com", referrer: "https://aiex.mosslab.hqmoss.com/sw.js"
2024/02/27 10:11:12 [error] 580955#580955: *231 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 101.133.147.20, server: aiex.mosslab.hqmoss.com, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3210/", host: "8.130.86.208"

**Comment by arvinxx** [comment]
GPT给的回答：
![LobeChat_随便聊聊_2024-02-27](https://github.com/lobehub/lobe-chat/assets/28616219/52d116e1-c376-4d5f-8d42-7f238445c0c7)
